src/sdk/pynni/nni/bohb_advisor/bohb_advisor.py

```diff
@@ -557,7 +557,8 @@ def handle_report_metric_data(self, data):
             Data type not supported
         """
         logger.debug('handle report metric data = %s', data)
-
+        if 'value' in data:
+            data['value'] = json_tricks.loads(data['value'])
```

Comment:
Is there any documentation affected?

Reply:
This is a key/value from `data`, which is changed from obj to string, I think this is not documented before. But we should BTW.